cafebabel/django_migrations.py

```diff
@@ -135,8 +141,10 @@ def aggregate_gallery_body(gallery):
     for image in gallery['images']:
         image_fields = image['fields']
         image_src = normalize_image(image_fields['image'])
-        body += f'<p><img src="{image_src}" alt="{image_fields["title"]}"></p>'
-        body += image_fields['body']
+        body += ('<figure>'
+                 f'<img src="{image_src}" alt="{image_fields["title"]}">'
+                 f'<figcaption>{image_fields["body"]}</figcaption>'
+                 '</figure>')
```

Comment:
Other suggestion
```python
body += f'''<figure>
               <img…
         '''
```
More a matter of preference I suppose; I leave this to your consideration